metricbeat/docs/modules/docker/container.asciidoc

```diff
@@ -7,7 +7,6 @@ This file is generated! See scripts/mage/docs_collector.go
 
 include::../../../module/docker/container/_meta/docs.asciidoc[]
 
-This is a default metricset. If the host module is unconfigured, this metricset is enabled by default.
```

Comment:
Are these changes intended? Maybe to happen on different PR?

Reply:
Hmm not sure what happened there but I manually fixed it. Good catch! Thank you!